packages/components/inputs/select-input/src/select-input.tsx

```diff
@@ -462,6 +464,9 @@ const SelectInput = (props: TSelectInputProps) => {
                     ),
                   }
                 : {}),
+              ...(props.appearance === 'filter' && {
```

Comment:
Moving these changes into https://github.com/commercetools/ui-kit/pull/2975 since they likely are geared towards modifying the select component for the filters component.